### ISSUE TITLE
WSL2 CUDA11.7 both docker installation and normal installation not working for instant-ngp

### ISSUE THREAD

**Issue by KamiCalcium** [nerfstudio-project/nerfstudio#1232]
**Env**

WSL2 + CUDA11.7
Can build tiny-cuda-dnn from the source code, also did
```
pip install git+https://github.com/NVlabs/tiny-cuda-nn/#subdirectory=bindings/torch
```
Try to run instant-NGP

**Under normal installation:**

Error message: 
```
FAILED: nerfacc_cuda.so 
c++ pybind.cuda.o render_weight.cuda.o pack.cuda.o cdf.cuda.o render_transmittance.cuda.o intersection.cuda.o render_transmittance_cub.cuda.o contraction.cuda.o ray_marching.cuda.o -shared -L/home/user/anaconda3/envs/nerfstudio/lib/python3.8/site-packages/torch/lib -lc10 -lc10_cuda -ltorch_cpu -ltorch_cuda_cu -ltorch_cuda_cpp -ltorch -ltorch_python -L/home/user/anaconda3/envs/nerfstudio/lib64 -lcudart -o nerfacc_cuda.so
/usr/bin/ld: cannot find -lcudart: No such file or directory
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
```
Tried
```
export LIBRARY_PATH="/usr/local/cuda/lib64:$LIBRARY_PATH"
```
and
```
export LIBRARY_PATH="/usr/local/cuda/lib64/stubs:$LIBRARY_PATH"
```
Does not work. Do have libcudart.so under /usr/local/cuda/lib64.

**Under Docker Image:**

Running:
```
sudo docker run --gpus all -v nerfstudio_docker:/workspace/ -v /home/sizhean/.cache/:/home/user/.cache/ -p 7007:7007 --rm -it dromni/nerfstudio:0.1.14 \
ns-train instant-ngp --data data/nerfstudio/poster
```
Error message:
```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/user/.local/bin/ns-train", line 8, in <module>
    sys.exit(entrypoint())
  File "/home/user/nerfstudio/scripts/train.py", line 248, in entrypoint
    main(
  File "/home/user/nerfstudio/scripts/train.py", line 232, in main
    config.save_config()
  File "/home/user/nerfstudio/nerfstudio/configs/experiment_config.py", line 120, in save_config
    base_dir.mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.10/pathlib.py", line 1179, in mkdir
    self.parent.mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.10/pathlib.py", line 1179, in mkdir
    self.parent.mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.10/pathlib.py", line 1179, in mkdir
    self.parent.mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.10/pathlib.py", line 1175, in mkdir
    self._accessor.mkdir(self, mode)
PermissionError: [Errno 13] Permission denied: 'outputs'
```
Looks like the inside the container it is not able to make new directory for the output?

Any help will be appreciate!!

**Comment by KamiCalcium** [comment]
Docker problem resolved. Need to change the relative path to absolute path when mounting the drive.
Normal installation's cuda issue still there.

**Comment by KamiCalcium** [comment]
nerfacc issue solved with nerfstudio 0.1.15 + nerfacc 0.3.3

**Comment by silver-obelisk** [comment]
> Docker problem resolved. Need to change the relative path to absolute path when mounting the drive. Normal installation's cuda issue still there.

I meet the same problem.Could you tell me how to solve it with more detail? thank you so much!

`Traceback (most recent call last):
  File "/usr/lib/python3.10/pathlib.py", line 1175, in mkdir
    self._accessor.mkdir(self, mode)
FileNotFoundError: [Errno 2] No such file or directory: 'outputs/poster/nerfacto/2023-06-15_173336'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.10/pathlib.py", line 1175, in mkdir
    self._accessor.mkdir(self, mode)
FileNotFoundError: [Errno 2] No such file or directory: 'outputs/poster/nerfacto'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.10/pathlib.py", line 1175, in mkdir
    self._accessor.mkdir(self, mode)
FileNotFoundError: [Errno 2] No such file or directory: 'outputs/poster'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/user/.local/bin/ns-train", line 8, in <module>
    sys.exit(entrypoint())
  File "/home/user/nerfstudio/nerfstudio/scripts/train.py", line 260, in entrypoint
    main(
  File "/home/user/nerfstudio/nerfstudio/scripts/train.py", line 244, in main
    config.save_config()
  File "/home/user/nerfstudio/nerfstudio/configs/experiment_config.py", line 126, in save_config
    base_dir.mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.10/pathlib.py", line 1179, in mkdir
    self.parent.mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.10/pathlib.py", line 1179, in mkdir
    self.parent.mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.10/pathlib.py", line 1179, in mkdir
    self.parent.mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.10/pathlib.py", line 1175, in mkdir
    self._accessor.mkdir(self, mode)
PermissionError: [Errno 13] Permission denied: 'outputs'
user@29d64aab6c12:/workspace$ ns-download-data nerfstudio --capture-name=poster
Traceback (most recent call last):
  File "/home/user/.local/bin/ns-download-data", line 8, in <module>
    sys.exit(entrypoint())
  File "/home/user/nerfstudio/nerfstudio/scripts/downloads/download_data.py", line 481, in entrypoint
    main(tyro.cli(Commands))
  File "/home/user/nerfstudio/nerfstudio/scripts/downloads/download_data.py", line 473, in main
    dataset.save_dir.mkdir(parents=True, exist_ok=True)
  File "/usr/lib/python3.10/pathlib.py", line 1175, in mkdir
    self._accessor.mkdir(self, mode)
PermissionError: [Errno 13] Permission denied: 'data'
`
